test/Microsoft.ML.Benchmarks/PredictionEngineBench.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System.IO;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Api;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class PredictionEngineBench
+    {
+        private PredictionEngine<IrisData, IrisPrediction> _irisModel;
+        private readonly string _irisModelPath = Program.GetInvariantCultureDataPath("iris_model.zip");
+        private readonly IrisData _irisExample = new IrisData()
+        {
+            SepalLength = 3.3f,
+            SepalWidth = 1.6f,
+            PetalLength = 0.2f,
+            PetalWidth = 5.1f,
+        };
+
+        private PredictionEngine<SentimentData, SentimentPrediction> _sentimentModel;
+        private readonly string _sentimentModelPath = Program.GetInvariantCultureDataPath("sentiment_model.zip");
+        private readonly SentimentData _sentimentExample = new SentimentData()
+        {
+            SentimentText = "Not a big fan of this."
+        };
+
+        private PredictionEngine<BreastCancerData, BreastCancerPrediction> _breastCancerModel;
+        private readonly string _breastCancerModelPath = Program.GetInvariantCultureDataPath("bc_model.zip");
+        private readonly BreastCancerData _breastCancerExample = new BreastCancerData()
+        {
+            Features = new[] { 5f, 1f, 1f, 1f, 2f, 1f, 3f, 1f, 1f }
+        };
+
+        [GlobalSetup(Targets = new string[] { nameof(MakeIrisPredictions), nameof(MakeSentimentPredictions), nameof(MakeBreastCancerPredictions)})]
+        public void SetupPipelines()
+        {
+            using (var env = new ConsoleEnvironment(seed: 1, conc: 1))
```

Comment:
Using this might lead to printing of some ML.net related stuff to the screen which will lead benchmarkDotNet to print bunch of parsing errors.
Solution will be to use something like https://github.com/Anipik/machinelearning/blob/9a6c384abc466053ffabc2edd34fd1aec3d22320/test/Microsoft.ML.Benchmarks/Numeric/Ranking.cs#L71